addons/web/static/src/js/views/control_panel/control_panel_model.js

```diff
@@ -795,7 +795,7 @@ var ControlPanelModel = mvc.Model.extend({
      */
     _getGroupIdOfType: function (type) {
         var self = this;
-        return Object.keys(this.groups).find(function (groupId) {
```

Comment:
same here

Reply:
here not, we need the ID and the find would return the object instead of the id. I would need to add an extra if to ensure there is a item found to returns its id, which is not nice.

Reply:
ok